docs/using-dev-sources-workflow.md

```diff
@@ -0,0 +1,65 @@
+---
+id: using-repo-sources-workflow
+title: Using Unreleased Packages
+---
+
+esy allows to use unreleased versions of packages hosted in their development
+repositories. This is useful as it allows to try new package versions before
+they are released.
+
+## With esy packages
+
+To use an unreleased version of an esy package specify a special type of
+dependency in [`resolutions`][cfg-resolutions] field:
+
+```json
+"resolutions": {
+  "reason": "facebook/reason"
+}
+```
+
+This will fetch `reason` package sources from [facebook/reason][] GitHub
```

Comment:
Looks like a missing link here

Reply:
It is defined in the "footnotes" style manner.